lib/xenops_server.ml

```diff
@@ -139,6 +139,8 @@ type vm_migrate_op = {
   ; vmm_vif_map: (string * Network.t) list
   ; vmm_vgpu_pci_map: (string * Pci.address) list
   ; vmm_url: string
+  ; vmm_tmp_src_id: Vm.id
+  ; vmm_tmp_dest_id: Vm.id
```

Comment:
@robhoes we talked about adding these fields to the `vm_migrate_op` record. My understanding is that this record is not actually sent over rpc (`VM_receive_memory` requests are sent instead); which implies we don't need to worry about the case where src/dest xenopsd are do/don't have this change

Alternatively we can just derive the tmp ids from `vmm_id` wherever we need them (but I think it's nicer this way)

Reply:
Yes, these operations are entirely local to a xenopsd instance. The `VM_receive_memory` isn't sent anywhere either. Look for `let do_request` to see the point where the source xenopsd sends the HTTP request to the destination to kick off the migration.